lib/feature_management.rb

```diff
@@ -1,12 +1,4 @@
 class FeatureManagement
-  ENVS_DO_NOT_DISPLAY_FAKE_BANNER = %w[
-    idp.staging.login.gov secure.login.gov
-  ].freeze
-
-  ENVS_WHERE_PREFILLING_OTP_ALLOWED = %w[
-    idp.dev.login.gov idp.pt.login.gov idp.dev.identitysandbox.gov idp.pt.identitysandbox.gov
-  ].freeze
-
   ENVS_WHERE_PREFILLING_USPS_CODE_ALLOWED = %w[
```

Comment:
I don't like hardcoding domains like this, so maybe if anybody wanted, I can make this change for USPS also?